manifest-updater/secrets/packetEncryption.go

```diff
@@ -0,0 +1,89 @@
+package secrets
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+const (
+	//packetDecryptionKeyMaxAge = 20 * 24 * time.Hour
+	packetDecryptionKeyMaxAge = 20 * time.Second
```

Comment:
I definitely don't think we want to rotate every 20 seconds!

Reply:
Yep! Fragments of testing.